tests/triangleObj_test.cpp

```diff
@@ -292,15 +262,19 @@ namespace objects
 
     TEST(TRIANGLEOBJ_METHOD, RayInsideTriangleDoesNotHit)
     {
+        std::vector<TriangleObj> objectList = {TriangleObj(Vec3(0, 0, 1), Vec3(0, 0, 0), Vec3(1, 0, 0)), TriangleObj(Vec3(0, 4, 1), Vec3(0, 4, 0), Vec3(1, 4, 0))};
 
+        size_t iteration = 0;
+        RayHitData hitData;
         Ray tempRay(Vec3(0.25, 0, 0.25), Vec3(0, 1, 0)); // Ray has origin inside object1
+        std::vector<bool> ExpectedResults = {false, true};
```

Comment:
Shouldn't the name start from lower-cased letter to match the style in the file?

Reply:
fixed